src/cloud-api-adaptor/test/provisioner/provision.go

```diff
@@ -536,6 +536,30 @@ func (p *CloudAPIAdaptor) Deploy(ctx context.Context, cfg *envconf.Config, props
 		return err
 	}
 
+	// TODO can we do this better with go code?
```

Comment:
I think so. I just found this example https://github.com/kubernetes/client-go/blob/master/examples/create-update-delete-deployment/main.go#L111 of using client-go to update a deployment.